ext/tk/lib/tk.rb

```diff
@@ -4460,7 +4460,7 @@ def __configinfo_core(slot = nil)
           __font_optkeys.each{|optkey|
             optkey = optkey.to_s
             fontconf = ret.assoc(optkey)
-            if fontconf && fontconf.size > 2
+            if fontconf&.size > 2
```

Comment:
This is wrong, since it will raise NoMethodError (size) if fontconf is `nil`. Same applies for many of these other refactorings, it's only OK if there is nothing after the call.